### PR TITLE
LPS-161893 | master

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-161893

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by liuxingye** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase [hunk 1]
```diff
@@ -112,6 +112,49 @@ definition {
 			objectName = "CustomObject184");
 	}
 
+	@description = "LPS-161893 - Verify that it's possible to export an object entry with aggregation field"
+	@priority = "4"
+	test CanExportObjectWithAggregationField {
+		task ("Create Custom Object and Custom Relationship itself (via API)") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase:118]: Given: A custom object with a self-relationship

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase [hunk 2]
```diff
@@ -112,6 +112,49 @@ definition {
...
+				objectName_1 = "CustomObject161893",
+				objectName_2 = "CustomObject161893",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject161893");
+		}
+
+		task ("When go to Custom Object > Fields tab and add Custom Aggregation Field (via UI)") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase:134]: When: An Aggregation field is added to the custom object from the Fields tab

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase [hunk 3]
```diff
@@ -112,6 +112,49 @@ definition {
...
+
+			ObjectAdmin.addObjectFieldViaUI(
+				field = "ID",
+				fieldLabel = "Custom Aggregation",
+				fieldType = "Aggregation",
+				function = "Max",
+				relationship = "Relationship");
+		}
+
+		task ("Then assert the object with aggregation field can be exported") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase:149]: Then: The custom object with a self-relationship and Aggregation field can be exported

**Review by timpak — CHANGES_REQUESTED** [review]
Hey @liuxingye, great job with this test! Can you update the wording for the Given When Then and forward this?

**Comment by liuxingye** [comment]
Thanks @timpak, please see https://github.com/liferay-bpm-qa/liferay-portal/pull/424

**Pull request closed without merging**